### PR TITLE
add note about broker properties restart

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2542
Review deadline:

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/properties/broker-properties.adoc [hunk 1]
```diff
@@ -6,6 +6,8 @@ Broker configuration properties are applied individually to each broker in a clu
 
 For information on how to edit broker properties, see xref:manage:cluster-maintenance/node-property-configuration.adoc[].
 
+NOTE: Modifying a broker property requires that you restart Redpanda for your changes to take effect.
```
Comment (micheleRP) [inline modules/reference/pages/properties/broker-properties.adoc]:
```suggestion
NOTE: All broker property require that you restart Redpanda for any update to take effect.
```

Comment (micheleRP) [reply modules/reference/pages/properties/broker-properties.adoc]: @Deflaimun Let's put a similar NOTE on the cluster properties page and the object storage properties page:

- NOTE: Some cluster properties require that you restart Redpanda for any update to be applied. 
- NOTE: Some object storage properties require that you restart Redpanda for any update to be applied. 

And is it correct that no topic properties require restart?

**Comment by Deflaimun** [comment]
Related conversation 
https://redpandadata.slack.com/archives/C04LYNJQUGN/p1718388434136009

File: modules/reference/pages/properties/broker-properties.adoc [hunk 2]
```diff
@@ -6,6 +6,8 @@ Broker configuration properties are applied individually to each broker in a clu
 
 For information on how to edit broker properties, see xref:manage:cluster-maintenance/node-property-configuration.adoc[].
 
+NOTE: All broker property require that you restart Redpanda for any update to take effect.
```
Comment (kbatuigas) [inline modules/reference/pages/properties/broker-properties.adoc]:
```suggestion
NOTE: All broker properties require that you restart Redpanda for any update to take effect.
```

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Just one typo to fix!

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**